lib/log.h

```diff
@@ -170,9 +170,9 @@ do {									\
 #endif
 
 #if defined(DEBUG) && (DEBUG >= 4)
-#define T_DBGV(...)	pr_debug(__BNR "#     " __VA_ARGS__)
+#define T_DBG4(...)	printk(__BNR "#     " __VA_ARGS__)
```

Comment:
Why do we need `#`? The debug messages look strange:
```
[   91.345848] [tempesta fw] 192.168.100.1 "default" "GET / HTTP/2.0" 200 0 "-" "h2load nghttp2/1.43.0"
[   91.346735] [tempesta fw] #     tfw_h2_sched_stream_dequeue: ctx [ffff88814fe04ca8] stream with id 13
```
We identify the debug level messages with identation

Reply:
I remove #. As I see the idea was to add extra two spaces for each debug level. I leave it as is. Should I change it?